controller/scheduler-v2/scheduler.go

```diff
@@ -498,7 +498,7 @@ func (s *Scheduler) HandleJobRequest(req *JobRequest) {
 func (s *Scheduler) RunPutJobs() {
 	log := logger.New("fn", "RunPutJobs")
 	log.Info("starting job persistence loop")
-	strategy := attempt.Strategy{Delay: 100 * time.Millisecond, Total: time.Minute}
+	strategy := attempt.Strategy{Delay: 100 * time.Millisecond, Total: time.Second}
```

Comment:
The controller client [uses a retry dialer](https://github.com/flynn/flynn/blob/scheduler-v2-review/controller/client/client.go#L134) which use a [1s connect timeout and 30s retry timeout](https://github.com/flynn/flynn/blob/scheduler-v2-review/pkg/dialer/retry.go#L10-L20), so having a 1s strategy here probably doesn't make sense.

It is actually quite important for these calls to eventually succeed (see #686), so maybe we need a pool of goroutines which are calling `PutJob`? We need to make sure they are not delivered out of order for any given job though (see [this DB constraint](https://github.com/flynn/flynn/blob/master/controller/schema.go#L132-L140)).